### DISCUSSION TITLE
How can I use camelCase for the file names (and not kebab-case) ?

### DISCUSSION THREAD

**Discussion by hgndgn** [infinitered/ignite#2133] [Q&A] [answered]
Always when I use the generator, the file names are in kebab-case. Is there a way to generate files name in camelCase?

**Comment by kateinkim** [comment, marked as answer]
Hi @hgndgn This is going to be changed from using kebabCaseName to pascalCaseName in our new Maverick release. However, we will look into this to see if we can update with more options to choose your file names. Thanks!

**Reply by hgndgn** [reply]
@kateinkim Thanks for the response! Do you know when the new release will come out?

**Reply by kateinkim** [reply]
Ignite version 8 "Maverick" beta is already out as of the 9th!  🎉 
If you'd like to test it out you can run `npx ignite-cli@next new PizzaApp`.  Also, we have a [PR](https://github.com/infinitered/ignite/pull/2139/files) for configuring file names in different cases but it won't be merged until after Maverick is officially released which will take a couple more weeks. 😄
